mlflow/xgboost.py

```diff
@@ -71,6 +71,7 @@
 from mlflow.tracking._model_registry import DEFAULT_AWAIT_MAX_SLEEP_SECONDS
 
 FLAVOR_NAME = "xgboost"
+FLAVOR_NAME_SKLEARN = "xgboost_sklearn"
```

Comment:
Use a new flavor to enable autologging for XGBoost sklearn estimators using `mlflow.sklearn._autolog()` but avoid flavor conflict with the `sklearn` flavor.

Reply:
Since `FALVOR_NAME_SKLEARN` is only used internally when activating XGBoost sklearn autologging, we can consider using it as a local variable instead of a global one.